app/src/pages/inside/settingsPage/settingsTabs/defectTypesTab/defectTypesTab.scss

```diff
@@ -0,0 +1,261 @@
+.settings-tab-content {
```

Comment:
Please split this file in different files (each component should have its' own scss file)